README.md

```diff
@@ -9,23 +9,26 @@ Reference application developed in the [Functional event-driven architecture: Po
 
 ## Table of contents
 
-* [Web App](#web-app)
-* [Overview](#overview)
-* [Requirements](#requirements)
-* [Services](#services)
-   * [Lib](#lib)
-   * [Domain](#domain)
-   * [Core](#core)
-   * [Feed](#feed)
-   * [Forecasts](#forecasts)
-   * [Processor](#processor)
-   * [Snapshots](#snapshots)
-   * [Alerts](#alerts)
-   * [WS Server](#ws-server)
-   * [Tracing](#tracing)
-   * [Tests](#tests)
-   * [X Demo](#x-demo)
-* [Monitoring](#monitoring)
+- [trading](#trading)
+  - [Table of contents](#table-of-contents)
+  - [Web App](#web-app)
+  - [Overview](#overview)
+  - [Requirements](#requirements)
+    - [Running application](#running-application)
```

Comment:
This addition is great but I would leave the rest as it was

Reply:
Annoyingly, my markdown plugin made this change automatically. I switched it back.